.github/workflows/dependencies.yml

```diff
@@ -12,8 +12,8 @@ jobs:
         id: credentials
         uses: actions/create-github-app-token@v2.0.2
         with:
-          app-id: ${{ secrets.GH_APP_ID }}
-          private-key: ${{ secrets.GH_APP_PRIVATE_KEY }}
```

Comment:
These tokens were for the kind `@slack-cli-releaser[bot]` of #5.

If these changes are alright I plan to **delete** these secrets to avoid future confusions 🫡